data/hails.txt

```diff
@@ -1664,6 +1781,13 @@ phrase "friendly pirate"
 		"around me"
 	word
 		"."
+		"."
+		"."
+		"!"
+		", if you don't want any trouble."
+		", if you were smart."
```

Comment:
you were -> you're

Reply:
Just because you can say "you're" doesn't mean you have to.

Reply:
That should be in present tense.

Reply:
indeed, although informal language seem to fit better here

Reply:
You guys have way too much hope that the pirates aren't totally incompetent.

Reply:
It's not a matter of personal style; the tenses are mismatched, as @Bladewood said.

Reply:
it's a pirate. you really think they can speak perfect english? it's not like anything is preventing them from using the words in this manner. Someone added a pirate that says "yer" and that's allowed? js. if it was anyone else it'd make sense to f ix it.

Reply:
the idea of pirate hails is more them using informal language and "slang", while still keeping the sentences themselves correct

Reply:
I imagine that being a pirate doesn't exclude the ability to speak perfect English. Also, you're is more natural than you where.
My advice is to try saying them out loud, if you sound like an idiot then you need to change something.

Reply:
Why not add both? I say you were and you are in normal conversation.